ActionSheet.ios.js

```diff
@@ -5,10 +5,20 @@ import { ActionSheetIOS, View } from 'react-native';
 
 export default class ActionSheet extends React.Component {
   render() {
-    return <View style={{ flex: 1 }}>{React.Children.only(this.props.children)}</View>;
+    return (
```

Comment:
any reason you changed the formatting here? Seems like the closing tag has an extra space. Do you mind running this through `prettier`?

Reply:
I reverted the change. I couldn't find any prettier settings anywhere so I'm not sure what prettier options to run it through. If you know what they are, I'll gladly run it.